src/db/database.py

```
@@ -252,5 +253,110 @@ def get_database_info() -> dict:
+            update_data["transcript_duration"] = transcript_duration
+        if transcript_confidence is not None:
+            update_data["transcript_confidence"] = transcript_confidence
+
+        # Update the episode in the database
+        with get_db_session() as session:
+            session.query(Episode).filter(Episode.uuid == uuid).update(
+                update_data,
+            )
+            session.commit()
+    except Exception as e:
+        raise e
```

Ruff (0.14.10)

Remove exception handler; error is immediately re-raised

(TRY203)